Week 1/Weekly Practice Problems (Week 1)/2413: Smallest Even Multiple/judah_silva.cpp

```diff
@@ -0,0 +1,9 @@
+class Solution {
+public:
+    int smallestEvenMultiple(int n) {
+        for (int i = n; ; i += n) {
+            if ((i % 2) == 0) return i;
+        }
+        return -1;
```

Comment:
The interesting thing to note about this problem is that the answer will always be either itself, or 2x itself (any odd number doubled will be even), so we can shorten this to one simple if statement.
```suggestion
        if (n%2 == 0) {
            return n;
        }
        return n*2;
```